app/javascript/crayons/MentionAutocompleteTextArea/MentionAutocompleteTextArea.jsx

```diff
@@ -287,6 +288,7 @@ export const MentionAutocompleteTextArea = ({
           as="textarea"
           autocomplete={false}
           onChange={handleTextInputChange}
+          data-active={hasSearchTerm}
```

Comment:
I'm using the `data-active` data attribute instead of the `disabled` attribute as it appeared to cause styling issues and weird behaviour.